packages/platform/src/batch.rs

```diff
@@ -142,6 +142,14 @@ impl Batch {
         res
     }
 
+    pub fn len(&self) -> usize {
+        self.msgs.len()
+    }
+
+    pub fn is_empty(&self) -> bool {
+        self.msgs.is_empty()
+    }
+
```

Comment:
please add unit tests to the `test` rust sub-module or at least add asserts to the existing unit tests